lib/octobox/configurator.rb

```diff
@@ -100,26 +88,23 @@ def sidekiq_schedule_path
     attr_writer :sidekiq_schedule_path
 
     def restricted_access_enabled
-      @restricted_access_enabled || ENV['RESTRICTED_ACCESS_ENABLED'].present?
+      @restricted_access_enabled || env_boolean('RESTRICTED_ACCESS_ENABLED')
     end
     attr_writer :restricted_access_enabled
 
     def github_organization_id
-      id = @github_organization_id || ENV['GITHUB_ORGANIZATION_ID']
-      return id.to_i if id.present?
+      @github_organization_id || env_integer('GITHUB_ORGANIZATION_ID')
     end
     attr_writer :github_organization_id
 
     def github_team_id
-      id = @github_team_id || ENV['GITHUB_TEAM_ID']
-      return id.to_i if id.present?
+      @github_team_id || env_integer('GITHUB_TEAM_ID')
     end
     attr_writer :github_team_id
 
     def native_link
       ENV['OCTOBOX_NATIVE_LINK'] || nil
     end
-    attr_writer :native_link
```

Comment:
Removed `attr_writer` for methods that won't currently respect those options if overridden, may end up adding them back in and enabling them in the config initializer.